### ISSUE TITLE
Deploy Dify as a Nocode AI assistant tool for RAG workflows

### ISSUE THREAD

**Issue by pascalwhoop** [everycure-org/matrix#124]
**Is your feature request related to a problem? Please describe.**

- We need a tool that allows us to extract online resources / information from the internet and augment e.g. a drug-disease pair with this information
- this tool should be minimal self development effort and instead leverage the existing GenAI hype wave and its developments
- Extensive market research has been done by @pascalwhoop in the last 12 months in the context of advising various enterprises on this topic and https://dify.ai/ kept convincing as a best-in-class offering

**Describe the solution you'd like**

- A deployment of https://dify.ai/ via helm https://github.com/douban/charts/tree/master/charts/dify 
- Web Access protected via IaP from Google / API accessible via cluster internal comms

**Describe alternatives you've considered**

Basically half the map of https://mad.firstmark.com/card#recdRNLWaU95VoEj3

A PoC is accessible here:
http://dify.home.curiloo.com/workflow/taKCDdcaDcg99uwM
